lib/src/dbs/executor.rs

```diff
@@ -260,7 +259,7 @@ impl<'a> Executor<'a> {
 					Ok(Value::None)
 				}
 				// Process param definition statements
-				Statement::Set(mut stm) => {
+				Statement::Set(stm) => {
```

Comment:
Perhaps we should change the `Statement::compute` to mut? It would make live queries easier as they modify themselves for permissions. Beyond the scope of this change though.

Reply:
There are lots of complexities with this. Mainly because compute can't be mut because it's concurrent. However this is something that will have to change in order to support foreach statements (which can mutate the context preceding the statement).